pillar/credere_production.sls

```diff
@@ -0,0 +1,28 @@
+apache:
+  sites:
+    credere:
+      servername: ocp24.open-contracting.org
+
+docker_apps:
+  credere_frontend:
+    target: credere-frontend
+    port: 8000
```

Comment:
Don't need to repeat port from _common (same in credere_dev and for credere_backend)